src/app/components/mnoe-api/notifications.svc.coffee

```diff
@@ -0,0 +1,21 @@
+# Service for managing the users.
+angular.module 'mnoEnterpriseAngular'
+  .service 'MnoeNotifications', (MnoeFullApiSvc, MnoeOrganizations) ->
+    _self = @
+
+    @get = (params = {})->
+      MnoeOrganizations.get().then(->
+        MnoeFullApiSvc
+          .one('organizations', MnoeOrganizations.getSelectedId())
+          .getList('notifications', params)
+      )
+
+    @notified = (params)->
+      MnoeOrganizations.get().then(->
+        MnoeFullApiSvc
+          .one('organizations', MnoeOrganizations.getSelectedId())
+          .one('notifications')
```

Comment:
If i used this syntax, the url called is `/mnoe/jpi/v1/organizations/41/notifications` instead of `"/mnoe/jpi/v1/organizations/41/notifications/notified`